test/evergreen.yml

```diff
@@ -4399,7 +4240,6 @@ buildvariants:
       WT_BUILDDIR=$WT_TOPDIR/cmake_build
       PATH=/cygdrive/c/Python39:/cygdrive/c/Python39/Scripts:$PATH
       PYTHONPATH=($WT_TOPDIR/lang/python/wiredtiger):$(cygpath -w $WT_TOPDIR/lang/python)
-    windows_configure_flags: -vcvars_bat "'C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\VC\Auxiliary\Build\vcvars64.bat'"
```

Comment:
What was this doing? It's not obvious to me that we can remove it.

Reply:
It was redundant and in the windows powershell script directly.

Reply:
Cool, thanks.